sycl/test-e2e/HostInteropTask/interop-task-hip.cpp

```diff
@@ -0,0 +1,175 @@
+// FIXME: the rocm include path and link path are highly platform dependent,
+// we should set this with some variable instead.
+// RUN: %{build} -o %t.out -I/opt/rocm/include -L/opt/rocm/lib -lamdhip64
+// RUN: %{run} %t.out
+// REQUIRES: hip
+
+#include <iostream>
+#include <sycl/sycl.hpp>
+
+#define __HIP_PLATFORM_AMD__
```

Comment:
Just a comment, not a blocker.
Should that be guarded in `#ifndef __HIP_PLATFORM_AMD__`? I think if this specific macro gets redefined it may not issue any warnings but I am not 100% on the redefinition rules (I think that one applies to object-like macros coming from C). Ideally, I'd normally still guard my `#defines` if I know the macro exists and the compiler could define it.

Reply:
Thanks @GeorgeWeb change made.